deployments/deploy-ssh.sh

```diff
@@ -346,6 +346,7 @@ if [ -n "${INTERNAL_DOCKER_REGISTRY}" ]; then
 fi
 
 kubectl create configmap cloud-config --from-file=/config/cloud-json
+kubectl create configmap cloud-source-config --from-file=/cluster/cloud-source-config
```

Comment:
Why do we need this?

Reply:
THis is for the spawn .. it uses different kubeconfig

Reply:
Why do we have the same thing in both `deploy-ssh.sh` as well as `deploy-pxb-cloud.sh`?
Do we need this for both on prem and cloud?